scripts/docker-start

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+./scripts/docker-wait-for-port mongo:27017 &&
+# Always run at least 2 processes for availability during process restarts. You can increase
+# this number if you wish to take advantage of more than 2 cores.
+pm2-runtime -i 2 start app.js
```

Comment:
Yes you can do `-i max`, but this could show one core which would lead to only 1, which is actually never good. No reason some conditional logic couldn't be written to cope with that. pm2-runtime might fall back to 2 for this reason, might be worth checking the source.

BTW pm2-runtime is an official sidekick built into pm2 for use as a docker runtime wrapper.